sdk/synapse/synapse/package.json

```
@@ -0,0 +1,58 @@
+{
+  "name": "@azure/synapse",
+  "author": "Microsoft Corporation",
+  "description": "SynapseClient Library with typescript type definitions for node.js and browser.",
+  "version": "1.0.0",
```

Is this meant to be a preview package?

---

> Is this meant to be a preview package?

Yes, could you please tell me what version should be?

---

For the preview libraries we've been working on, we append `-preview.#`, so in this case:
```suggestion
  "version": "1.0.0-preview.1",
```